src/Test.UnitTests.BinSkim.Driver/DriverUtilitiesTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.IL;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.BinSkim.Rules
+{
+    public class DriverUtilitiesTests
+    {
+        [Fact]
+        public void GetInaccessibleSymbolPathsTests()
```

Comment:
@shaopeng-gh This isn't really a unit test right? Will these result in actual network calls? If yes, then you need to mock the `HttpClient` to make this a unit test. We need separate integration tests to ensure that network calls work properly, but again the authentication model is unclear to me.